src/lib/rmt/ssl/current_ca_password_dialog.rb

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2018 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require 'rmt/utils'
+require 'ui/dialog'
+
+module RMT; end
+module RMT::SSL; end
+
+class RMT::SSL::CurrentCaPasswordDialog < UI::Dialog
+  def initialize
+    textdomain 'rmt'
+
+    @cert_generator = RMT::SSL::CertificateGenerator.new
+  end
+
+  def dialog_content
+    VBox(
+      VSpacing(1),
+      Heading(_('Your CA private key is encrypted. Please input password')),
+      VSpacing(1),
+      HBox(
+        HSpacing(2),
+        VBox(
+          MinWidth(15, Password(Id(:ca_password), _('&Password')))
```

Comment:
It is very helpful for the translators to know some context for the translated messages, otherwise the translation might not be correct. Ideally add a `TRANSLATORS:` comment for each translated text.

In this case:

    # TRANSLATORS: Label for a password input field

Reply:
not sure about that. the way I implemented it now - we pass text labels for fields in variables and I think we should find better way for translating